app/assets/javascripts/admin/addon/components/schema-theme-setting/editor.gjs

```diff
@@ -180,11 +180,18 @@ export default class SchemaThemeSettingNewEditor extends Component {
   @action
   addChildItem(propertyName, parentNodeIndex) {
     this.activeData[parentNodeIndex][propertyName].pushObject({});
+
+    this.onChildClick(
+      this.activeData[parentNodeIndex][propertyName].length - 1,
```

Comment:
Is there any chance `this.activeData` doesn't return an array of array?

Reply:
It'll always be an array because the backend will validate for it.